src/container/container_iv.c

```diff
@@ -511,8 +511,15 @@ cont_iv_ent_fetch(struct ds_iv_entry *entry, struct ds_iv_key *key,
 					}
 					prop_entry = daos_prop_entry_get(prop, DAOS_PROP_CO_STATUS);
 					D_ASSERT(prop_entry != NULL);
-
 					daos_prop_val_2_co_status(prop_entry->dpe_val, &stat);
+
+					rc = ds_cont_tgt_open(entry->ns->iv_pool_uuid,
+							      civ_key->cont_uuid, chdl.ch_cont,
+							      chdl.ch_flags, chdl.ch_sec_capas,
+							      stat.dcs_pm_ver);
```

Comment:
I just not quite understood why this change could help to resolve the problem.
Because inside cont_iv_ent_refresh(just call cont_iv_ent_update) it will call ds_cont_tgt_open(). The cont_iv_ent_refresh() suppose will be called right after cont_iv_ent_fetch() on the root node during the processing of cont_iv_fetch(..., IV_CONT_CAPA,...). 
any difference for the calling in the two places? thx

Reply:
If I'm not mistaken, `ivo_on_refresh` is not called on the root node. My reading is that `crt_iv_fetch` will get 0 from `ivo_on_fetch`, and return quickly. Is that not the case?

Reply:
seems you are right, crt_hdlr_iv_fetch_aux() it will not call ivo_on_refresh() on root node.
Then it can explain why this change made difference for the test.